style/app.scss

```diff
@@ -687,131 +642,121 @@ a:active
 
 #sidebar
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  display          : flex;
+  flex-direction   : column;
+  z-index          : 1;
+  position         : inherit;
+  width            : 140px;
+  background-color : #ffffff;
+  border-radius    : 20px;
+  box-shadow       : 2px 2px 2px #e8e9eb;
+  margin-top       : 20px;
 }
 
-#sidebar-button
+.expanded
 {
-  cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #5c497e;
-  position         : absolute;
-  border           : solid 1px #000000;
+  visibility : visible;
 }
 
-#sidebar-button:hover
+.expanded > .sidebar-dropdown
```

Comment:
Change this selector to just `.sidebar-dropdown` (since you will be removing the height rule, see below comment).